ar5523: remove unused variable

struct ieee80211_tx_info *info isn't used anywhere in ar5523_tx().

Changes-licensed-under: ISC
Signed-off-by: Stefan Lippers-Hollmann <s.l-h@gmx.de>

--- a/drivers/staging/ar5523/ar5523.c
+++ b/drivers/staging/ar5523/ar5523.c
@@ -823,7 +823,6 @@ static int ar5523_tx(struct ieee80211_hw
 	struct ar5523 *ar = hw->priv;
 	struct ar5523_tx_data *data;
 	struct ar5523_tx_desc *desc;
-	struct ieee80211_tx_info *info = IEEE80211_SKB_CB(skb);
 	struct urb *urb;
 	int paylen = skb->len;
 	int error = 0;
